projects/core/src/lib/colors/color-palette-editor/color-palette-editor.component.html

```diff
@@ -0,0 +1,13 @@
+<cdk-virtual-scroll-viewport appendOnly itemSize="80" class="tabs-viewport">
+    <div *cdkVirtualFor="let c of allColors; let i = index" class="grid-container">
+        <geoengine-color-attribute-input
+            class="grid-colorpicker"
+            [ngModel]="c"
+            (ngModelChange)="rebuildColorMap(i, $event)"
+        ></geoengine-color-attribute-input>
+        <button mat-icon-button (click)="removeColorTab(i)" class="grid-removebutton"><mat-icon>remove</mat-icon></button>
```

Comment:
![grafik](https://user-images.githubusercontent.com/4947061/233296063-26327dc6-5976-4a79-98bc-b79ab9585dab.png)

Please add a minor gap between the delete button and the scrollbar. Users could accidentally click it when scrolling.

Reply:
Added a small gap in [72fbede](https://github.com/geo-engine/geoengine-ui/pull/614/commits/72fbede1946536060d858e548db2dec7318619d7)

Reply:
![grafik](https://user-images.githubusercontent.com/4947061/233667730-3785b62c-44b3-4149-982d-4a9f05a8f88d.png)

PCT became a rather large gap. Please use 0.25rem or 0.5rem instead.